src/meta.py

```
@@ -493,10 +495,8 @@ def __init__(self, _data: dict[str, Any] | None = None, **kwargs: Any) -> None:
             setattr(self, k, v)
 
     def copy(self) -> Meta:
```

Ruff (0.15.20)

[error] Undefined name `Meta`

(F821)